### DISCUSSION TITLE
request to add fastify support

### DISCUSSION THREAD

**Discussion by ingen2** [rubiin/ultimate-nest#528] [Ideas]
### Clear and concise description of the problem

[Fastify](https://www.fastify.io/) is a high performance and low overhead framework highly focused on providing maximum efficiency and speed. Read how to use it [here](https://docs.nestjs.com/techniques/performance).

### Suggested solution

please add fastify support for better http performance

### Validations

- [X] Follow our [Code of Conduct](https://github.com/rubiin/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/rubiin/.github/blob/main/CONTRIBUTING.md).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by rubiin** [comment]
Adding fastify will introduce a major rewrite as the stuffs we are using works in different way and is not a drop in replacement for example `passport` and `fastify-passport`. However I will leave this as a discussion so I may be able to go through the rewrite considering a number of people want to go in that direction
